README.md

```diff
@@ -3,8 +3,13 @@
 This repository serves as a check-in for attendees of the **Git Workshop for Modern Software Engineers** hosted by Basti Ortiz, the Director for Engineering of UP CSI.
 
 ---
+
 ## Attendees
 
+Marius Barcenas

```

Comment:
Let's just add a tiny correction here. I'll let you commit the suggestion yourself.
```suggestion
* Marius Barcenas
```